### PR TITLE
Display shadow element when dragging from notebook

### PR DESCRIPTION
**Pull request by fcollonval**
This is a follow-up of #100

It inserts a shadow widget when dragging a cell from a notebook or moving the existing the widget.

### CODE REVIEW

**Comment by hbcarlos** [comment]
Looks great, thanks!

File: packages/jupyterlab-gridstack/package.json [hunk 1]
```diff
@@ -43,8 +43,8 @@
     "watch:src": "tsc -w"
   },
   "dependencies": {
-    "@jupyter-widgets/base": "^4.0.0",
-    "@jupyter-widgets/jupyterlab-manager": "^3.0.0",
+    "@jupyter-widgets/base": "^4.0.0-alpha.2",
```
Comment (jtpio) [inline packages/jupyterlab-gridstack/package.json:46]: Was this change intentional? Looks like it reverts some parts of https://github.com/voila-dashboards/voila-gridstack/pull/100

Comment (fcollonval) [reply packages/jupyterlab-gridstack/package.json:46]: Good catch - I started over from reverting a commit a miss that one. I open #112 to redo the changes.
